### ISSUE TITLE
Static class name dynamically

### ISSUE THREAD

**Issue by b4-tahayashi** [A1rPun/vue-quick-edit#2]
Some class names are given as props, but `.vue-quick-edit-clickable` , `.vue-quick-edit-error`, `.vue-quick-edit-empty`  remain static classes.

If I want to change the classes and appearance, .vue-quick-edit is left inconvenient.

I want to be able to inject a classes.
https://github.com/A1rPun/vue-quick-edit/blob/master/src/quick-edit.vue#L90-L92

**Comment by A1rPun** [comment]
Thanks a lot! This will get implemented :)

**Comment by A1rPun** [comment]
As requested these are now configurable, I should be able to release this today:
```
isClickable: 'vue-quick-edit__link--is-clickable'
isEmpty: 'vue-quick-edit__link--is-empty'
isRequired: 'vue-quick-edit__link--is-required'
```

**Comment by b4-tahayashi** [comment]
Awesome!

**Comment by A1rPun** [comment]
:airplane: Published in `v1.2.3`
